src/sections/home/Hero.jsx

```diff
@@ -1,7 +1,32 @@
+import Button from "/src/components/button/Button";
+
 function Hero() {
+  const scrollToContact = () => {
+    document.getElementById("contact-form")?.scrollIntoView({
+      behavior: "smooth",
+      block: "start",
+    });
+  };
+
   return (
     <section>
-      <h2>Hero</h2>
+      <div className="flex flex-col baseline-start gap-4">
+        <img src="/src/assets/placeholder-img.png" alt="placeholder image" />
+        <h1 className="font-[var(--h1-size)] text-[var(--color-primary-nightwing-hex)] text-pretty text-left xs:text-center sm:text-center">
```

Comment:
The light dark toggle doesn't work with this. When using the AIs with tailwind 4 you should pay attention to how they add the CSS and classes, they don't having the docs in their knowledge base yet.
```suggestion
        <h1 className="font-h1-size text-color-primary-nightwing-hex text-pretty text-left xs:text-center sm:text-center">
```

Reply:
See #44 I've suggested some changes that we can merge and pull into this to stream like this would become something like this.
```jsx
<h1 className="text-h1 text-fg text-pretty text-left xs:text-center sm:text-center">
```

Reply:
thank you, updating the files now and will pay attention to this going forward.